recipes/pygad/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pygad" %}
+{% set version = "2.18.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/p/pygad/pygad-{{ version }}.tar.gz
+    sha256: a5f03a08368f6960fe9de8da870264190309567beb7084c017258d5fdc6557ec
+    patches:
+      - 152.patch
+
+  # This is just used to grab the LICENSE file, missing in the PyPI source archive
+  - url: https://github.com/ahmedfgad/GeneticAlgorithmPython/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 007e14d60d16379cdf4cce64bdf90507c21e216854c5b87fc25ec40594e31f1f
+    folder: github
+
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - matplotlib-base
+
+test:
+  imports:
+    - pygad
+  commands:
+    - pip list --verbose
+    - python -c "import pygad"
```

Comment:
This is the same as the line above under `imports`:
```suggestion
```

Reply:
Thanks! Indeed it was a debugging leftover.